onnxruntime/core/framework/execution_frame.cc

```diff
@@ -340,7 +343,8 @@ Status ExecutionFrame::AllocateMLValueTensorPreAllocateBuffer(OrtValue& ort_valu
 
     // be generous and use the buffer if it's large enough. log a warning though as it indicates a bad model
     if (buffer_num_elements >= required_num_elements) {
-      LOGS(session_state_.Logger(), WARNING) << message;
+      // Todo: View Operator is reusing the buffer bigger than the required size. Disabling warming message for now.
+      //LOGS_DEFAULT(WARNING) << message;
     } else {
```

Comment:
Is the View operator causing a warning something that will be resolved?

Can we limit disabling the warning to just a build with training enabled? 

Also please don't lose the change to use session_state_.Logger()